hardhat.config.ts

```diff
@@ -1,5 +1,42 @@
 import "@nomiclabs/hardhat-waffle";
 
+import dotenv from "dotenv";
+import type { HttpNetworkUserConfig } from "hardhat/types";
+import yargs from "yargs";
+
+const argv = yargs
+  .option("network", {
+    type: "string",
+    default: "hardhat",
+  })
+  .help(false)
+  .version(false).argv;
+
+// Load environment variables.
+dotenv.config();
+
+const DEFAULT_MNEMONIC =
+  "candy maple cake sugar pudding cream honey rich smooth crumble sweet treat";
+
```

Comment:
ubernit: It would be nice to destructure the environment once, so its easier to see in one place what all of the possible environment variables are:
```ts
const {
  INFURA_KEY,
  MNEMONIC,
  PK,
} = process.env;
```

Reply:
This would also shorten some of you if statements!